certgen/qrcode-image-generator/src/main/assembly/src.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove this file altogether...maynot be needed. Same case for other .properties file too.